bigquery_etl/glean_usage/templates/baseline_clients_first_seen_v1.sql

```diff
@@ -0,0 +1,97 @@
+{{ header }}
+
+{% if init %}
+
+CREATE TABLE IF NOT EXISTS
+  `{{ first_seen_table }}`
+PARTITION BY
+  submission_date
+CLUSTER BY
+  sample_id
+OPTIONS
+  (require_partition_filter = TRUE)
+AS
+SELECT
+  DATE(submission_timestamp) as submission_date,
+  sample_id,
+  client_info.client_id
+FROM
+  `{{ baseline_table }}`
+WHERE
+  FALSE
+
+{% else %}
+
+WITH
+{% if fennec_id %}
```

Comment:
I had imagined crafting the Fenix and Firefox for iOS queries by hand in the normal `sql/` tree, but encoding all of this in one place as you're doing is more understandable and handles the dataset-per-channel problem nicely. So I like this.